pkg/network/protocols/tls/gotls/testutil/helpers.go

```diff
@@ -0,0 +1,30 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build linux_bpf && test
+
+package testutil
+
+import (
+	"github.com/DataDog/gopsutil/host"
+
+	"github.com/DataDog/datadog-agent/pkg/network/config"
+	"github.com/DataDog/datadog-agent/pkg/network/protocols/http"
+)
+
+// isFedora returns true if the current OS is Fedora.
+// go-tls does not work correctly on Fedora 35, 36, 37 and 38.
+func isFedora() bool {
+	info, err := host.Info()
+	if err != nil {
+		return false
```

Comment:
You will return false instead of the error, and you would not know if you were able to get the info, or it is version that is different from the options you checked.

Reply:
changed to match https://github.com/DataDog/datadog-agent/pull/22199/files#diff-1b200ed6e29617e4a690bf1fbaf8694c2420b81bfa968c1a7ae221f36323c293R769-R775